tests/CoreCLR/Test.csproj

```diff
@@ -11,6 +11,24 @@
   <ItemGroup>
     <!-- Some tests consist of multiple assemblies - make sure ILC sees them -->
     <IlcCompileInput Include="$(MSBuildProjectDirectory)\*.dll" />
+
+    <!-- For Ready To Run testing, use the CoreCLR framework assemblies instead of the AOT ones -->
+    <IlcReference Condition="'$(NativeCodeGen)' == 'readytorun'" Include="$(CoreRT_CoreCLRRuntimeDir)\*.dll"
+      Exclude="$(CoreRT_CoreCLRRuntimeDir)\api-ms-win*.dll;
```

Comment:
Does anything break if we just pass these DLLs to ILC? AFAIK, nothing is going to look at unreferenced references and it would be nice not to include this fragile-looking list in two places.

Reply:
Nothing breaks. I added this to ward off feedback that I'm passing in a bunch of garbage to ILC. It makes the response files uglier but other than that, benign. I can revert these two blobs.